# HG changeset patch
# User Michael Gerdts <mike.gerdts@oracle.com>
# Date 1412623238 25200
#      Mon Oct 06 12:20:38 2014 -0700
# Node ID 08d5c81201b3215699b91cdc9b17f60c4bd86e7e
# Parent  1debb63439545fd0b30153eb68e884623d06c531
do not link with libpthread - developed by Oracle
Not submitted upstream: specific to Solaris but no Solaris Makefile upstream

diff -r 1debb6343954 -r 08d5c81201b3 Makefile
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 CFLAGS=-O3 -Wall -Wextra
 
 pigz: pigz.o yarn.o
-	$(CC) -o pigz pigz.o yarn.o -lpthread -lz
+	$(CC) -o pigz pigz.o yarn.o -lz
 	ln -f pigz unpigz
 
 pigz.o: pigz.c yarn.h
